### PR TITLE
Reliability: Flow types for tests (part 3)

### PR DESCRIPTION
**Pull request by alshdavid**
[Continuing from here](https://github.com/atlassian-labs/atlaspack/pull/277)

### CODE REVIEW

File: packages/core/integration-tests/test/globals.js [hunk 1]
```diff
@@ -58,7 +59,6 @@ describe('globals', function () {
     it('when scope hoisting is disabled', async function () {
       let bundleGraph = await bundle(path.join(dir, 'index.js'), {
         defaultTargetOptions: {
-          context: 'browser',
```
Comment (alshdavid) [inline packages/core/integration-tests/test/globals.js:61]: This is not on `InitialOptions` and does not appear to be used within core

File: packages/core/integration-tests/test/globals.js
Comment (alshdavid) [inline packages/core/integration-tests/test/globals.js:1]: `defaultTargetOptions.context` is not on `InitialOptions` and does not appear to be used within core

File: packages/core/integration-tests/test/html.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import assert from 'assert';
+// @flow
+import assert from 'assert/strict';
```
Comment (alshdavid) [inline packages/core/integration-tests/test/html.js:2]: Using local assert definition to add in `assert.rejects`

File: packages/core/integration-tests/test/html.js [hunk 3]
```diff
@@ -2647,7 +2674,10 @@ describe('html', function () {
 
       for (let htmlBundle of b.getBundles().filter((b) => b.type === 'html')) {
         let htmlSiblings = b
-          .getReferencedBundles(htmlBundle, true)
+          .getReferencedBundles(htmlBundle, {
+            recursive: true,
+            includeInline: true,
+          })
```
Comment (alshdavid) [inline packages/core/integration-tests/test/html.js:2680]: I think this is what it was intending to do?

File: flow-typed/node/assert.js
Comment (alshdavid) [inline flow-typed/node/assert.js:1]: flow types missing for `assert.rejects`, using override to pass type check

File: packages/core/integration-tests/test/html.js [hunk 4]
```diff
@@ -2317,7 +2333,7 @@ describe('html', function () {
       });
 
       // could run in either order.
-      assert(output.sort(), ['a', 'b', 'c']);
```
Comment (alshdavid) [inline packages/core/integration-tests/test/html.js:2320]:
```javascript
assert(['a', 'b', 'c', 'c'], ['a', 'b', 'c'])
```
Was erroneously passing. I have updated the assertion to include the missing asset. From the `assertBundles` above, my change looks correct but I'm not 100% sure. Please confirm that it's the expected output

**Review by yamadapc — APPROVED** [review]

**Pull request merged**